src/core/ItemData.cpp

```diff
@@ -246,13 +274,14 @@ int CItemData::WriteCommon(PWSfile *out) const
 {
   int i;
 
-  const FieldType TextFields[] = {GROUP, TITLE, USER, PASSWORD,
+  const FieldType TextFields[] = {GROUP, TITLE, USER, PASSWORD, TWOFACTORKEY,
```

Comment:
Would it make more sense to store the key as a binary value, assuming it would be used to calculate the TOTP more often than displaying it?